apps/namadillo/src/atoms/staking/services.ts

```diff
@@ -159,3 +160,7 @@ export const createClaimAndStakeTx = async (
     account.address
   );
 };
+
+export const clearClaimRewards = (accountAddress: string): void => {
+  queryClient.setQueryData(["claim-rewards", accountAddress], () => ({}));
```

Comment:
Did you try this as `invalidateQueries` https://tanstack.com/query/latest/docs/framework/react/guides/query-invalidation ? I think this is the "correct way" to clear queries for react-query

Reply:
Actually not. I'm overriding the result of the claim-rewards query, so no rewards are available :) . But maybe I can be more explicit on this. Will make a small change before merging